aptos-move/move-examples/post_mint_reveal_nft/sources/minting.move

```diff
@@ -378,12 +385,15 @@ module post_mint_reveal_nft::minting {
 
         let i = 0;
         while (i < vector::length(&token_uris)) {
+            let token_uri = vector::borrow(&token_uris, i);
+            assert!(!bucket_table::contains(&collection_config.dedupped_tokens, token_uri), error::invalid_argument(EDUPLICATE_TOKEN_URI));
             big_vector::push_back(&mut collection_config.tokens, TokenAsset {
-                token_uri: *vector::borrow(&token_uris, i),
+                token_uri: *token_uri,
                 property_keys: *vector::borrow(&property_keys, i),
                 property_values: *vector::borrow(&property_values, i),
                 property_types: *vector::borrow(&property_types, i),
             });
+            bucket_table::add(&mut collection_config.dedupped_tokens, *token_uri, true);
```

Comment:
do we imagine `add_tokens` will be called multiple times?
if so this is good
if not, then we probably can use this and then delete its contents... so we don't pay for storage.

Reply:
yeah I think it might be called multiple times since sometimes adding all the tokens at once exceeds the transaction size limit. this is a good point, I'll update this PR and delete the content when the user exchanges for the NFT.